native/src/routes/Offers.tsx

```diff
@@ -13,19 +13,21 @@ type OffersProps = {
   languageCode: string
 }
 
+const internalOffers = [SPRUNGBRETT_OFFER_ROUTE, MALTE_HELP_FORM_OFFER_ROUTE] as string[]
+
 const Offers = ({ offers, navigateToOffer, languageCode }: OffersProps): ReactElement => {
   const { t } = useTranslation('offers')
 
   const tiles = offers.map(offer => {
     let path = offer.path
-    if (offer.alias === SPRUNGBRETT_OFFER_ROUTE) {
+    if (offer.alias in [SPRUNGBRETT_OFFER_ROUTE, MALTE_HELP_FORM_OFFER_ROUTE]) {
       path = offer.alias
     }
     return new TileModel({
       title: t(offer.title),
       path,
       thumbnail: offer.thumbnail,
-      isExternalUrl: offer.alias !== SPRUNGBRETT_OFFER_ROUTE,
+      isExternalUrl: !internalOffers.includes(offer.alias),
```

Comment:
Optional: Either use `.includes` or `offer.alias in` for both `isExternalUrl` and `path` (which could be inlined). Like this its a little confusing and I thought its doing two different things.